src/signup.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>SUSI Login Portal</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="css/signup.css">
+</head>
+<body>
+    <div class="container">
+        <div class="row">
+            <div class="col-md-2">
+            </div>
+            <div class="col-md-8">
+                <div class="card">
+                    <img src="images/susi-logo.png" class="img-fluid signin-image">
+                    <div id="notsignup">
+                        <form id="signupForm">
+                            <div class="form-group">
+                                <label for="email">Email address:</label>
+                                <input type="email" class="form-control" id="email" placeholder="Email">
+                            </div>
+                            <br />
+                            <div class="form-group">
+                                <label for="password">Password:</label>
+                                <input type="password" class="form-control" id="password" placeholder="Enter your password">
+                            </div>
+                            <br />
+                            <button type="submit" class="btn btn-success" id="signupbutton">Signup</button>
+                            <br>
+                            <p style="text-align: center;">
+                                <a href="login.html" style="color: #fff; ">Already Registered?Want to Login</a>
```

Comment:
Could you add a space between "?" or add a br tage

Reply:
Check it now @stealthanthrax